### DISCUSSION TITLE
CAP_NET_RAW Privilege

### DISCUSSION THREAD

**Discussion by lisbethseldon** [projectdiscovery/naabu#668] [Q&A]
When I try to use script I get this message and it just finishes.

[INF] Current naabu version 2.1.6 (latest)
[INF] Running host discovery scan
[INF] Running SYN scan with CAP_NET_RAW privileges

I installed library, go, naabu again and again. But, I still get the same message.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
